src/pages/Home/Content/Content.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ContentStyle } from "./ContentStyle";
+import { FoodItem, Category } from "DummyData/DataTypes";
+import { McSpicy } from "DummyData/McSpicy";
+import { OvaltineMcFlurry } from "DummyData/OvaltineMcFlurry";
+import { Categories } from "DummyData/Categories";
+import SectionHeader from "./Section/SectionHeader/SectionHeader";
+import ItemList from "./Section/ItemList/ItemList";
+
+const AllFoodItems: FoodItem[] = [
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+];
+
+//trying to render 6 cards of identical McSpicy foodItems. Should be under the "recommended" SectionHeader.
+//McSpicy would appear both under "Recommended" and "Main Course". FoodItem can have more than one category.
+export default function Content() {
+  return (
+    <ContentStyle>
+      {Categories.map((sectionHeader) => {
+        const foodItemToRender: FoodItem[] = [];
+        AllFoodItems.forEach((foodItem) => {
+          foodItem.categories.forEach((foodItemCategory) => {
+            if (sectionHeader.name === foodItemCategory.name) {
+              foodItemToRender.push(foodItem);
+            }
+          });
+        });
+
+        return (
+          <>
```

Comment:
use the index of the map function to pass as the key

Reply:
Done. This introduced a bit of complexity because there is nested `forEach()`, but it works in the end.

Reply:
Logic to get the data to render should be placed outside of the return(). So the map and forEach should be outside. The resultant array of data is then mapped to create the component in the return(). As me if you don't get what i mean